coordinator/internal/orm/chunk.go

```diff
@@ -312,6 +316,8 @@ func (o *Chunk) InsertChunk(ctx context.Context, chunk *encoding.Chunk, dbTX ...
 		ProvingStatus:                int16(types.ProvingTaskUnassigned),
 		TotalAttempts:                0,
 		ActiveAttempts:               0,
+		CrcMax:                       0, // CrcMax is not supported in codecv0
```

Comment:
It's supported. but we can delete these two lines because this function (in coordinator) is only used in unit tests. and removing them would set default 0 (when initializing the golang type) in DB.